gempy/core/interpolator.py

```diff
@@ -611,12 +677,14 @@ def reset_flow_control_initial_results(self, reset_weights=True, reset_scalar=Tr
         Returns:
             True
         """
-        n_series = self.len_series_i.shape[0]#self.additional_data.get_additional_data()['values']['Structure', 'number series']
-        x_to_interp_shape = self.grid.values_r.shape[0] + 2 * self.len_series_i.sum()
+        n_series = self.len_series_i.shape[
+            0]  # self.additional_data.get_additional_data()['values']['Structure', 'number series']
```

Comment:
remove commented out block, allows removal of new line

Reply:
done